docs/best-practices/git-standardization.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Git Standardization
+---
+Document is going to contains the different standards relate to the use of Git within Mobile Jazz. 
+
+## Topics
+* [Commit messages](https://harmony.mobilejazz.com/docs/best-practices/git-standardization#commit-messages)
+* [Merge / Pull requests](https://harmony.mobilejazz.com/docs/best-practices/git-standardization#merge--pull-requests)
+* [Git methodology](https://harmony.mobilejazz.com/docs/best-practices/git-standardization#git-methodology)
+
```

Comment:
@joselufo only to double-check, this TOC is auto-generated with https://docusaurus.io/docs/markdown-features/toc ?
Also, by default Docusaurus adds a TOC on the right part, ex. https://harmony.mobilejazz.com/docs/best-practices/presenter-specs